### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by yaseerokino**
### Project requirements 

- The queries below take too much time (**1 sec = 1000ms can be considered as too much time for database query**). 
  - `SELECT COUNT(*) FROM visits where animal_id = 4;`
  - `SELECT * FROM visits where vet_id = 2;`
  - `SELECT * FROM owners where email = 'owner_18327@mail.com';`
- Used `EXPLAIN ANALYZE` on the above queries to check their execution time. 
- Find a way to decrease the execution time of the first query. Look for hints in the previous lessons.
- Find a way to improve the execution time of the other two queries.

@yaseerokino  and @mbdesigns1989 

BEFORE
 
owner
AFTER

owner-audit

BEFORE
visits

AFTER
visits1

BEFORE
visitors

AFTER
visitors-audi

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -66,4 +66,16 @@ CREATE TABLE visits (
     DATE DATE,
     CONSTRAINT FK_ANIMALS FOREIGN KEY(ANIMALS_ID) REFERENCES animals(ID),
     CONSTRAINT FK_VETS FOREIGN KEY(VETS_ID) REFERENCES vets(ID)
-);
```
Comment (SajjadAhmad14) [inline schema.sql:69]: The execution time in the screenshot before adding an index on the foreign key of `SELECT * FROM visits where vet_id = 2;` is far greater than 1000ms and it is actually 4276.9ms. Please make sure that the execution time should be around 1 sec (1000 ms) before adding indexes on foreign keys.

The execution time in the screenshot BEFORE adding an index on foreign key for `SELECT * FROM owners where email = 'owner_18327@mail.com';` is 756.9ms which is ower lower than 1 sec (1000ms). Please run the populate database query a few times more as described [here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=4%20Depening%20on,bigger%20than%201000ms.) in the project requirements and make sure the execution time is around 1000ms.

Please add all the screenshots for BEFORE and AFTER to the description of the PR.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi team, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and description. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema.sql [hunk 2]
```diff
@@ -1,69 +1,83 @@
 /* Database schema to keep the structure of entire database. */
+CREATE DATABASE vet_clinic;
+
```
Comment (addod19) [inline schema.sql:3]: - [ ] It would be better if we add the before and after screenshot so we can compare the execution time

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @yaseerokino and @mbdesigns1989, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 2 above)
Comment (mbdesigns1989) [reply schema.sql:3]: I added the before and after screenshot to the pull request description.

**Review by ElijahTobs — APPROVED** [review]
# STATUS: APPROVED 🎉 🥇 
## Hi @yaseerokino and @mbdesigns1989 , 

Good job making the changes required by the previous Code Reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

_Reviewed by_: __**Elijah Ayandokun**__

**Pull request merged**